e2e/charts/newrelic-infrastructure/templates/daemonset-kubelet.yaml

```diff
@@ -98,6 +100,10 @@ spec:
                 fieldRef:
                   apiVersion: "v1"
                   fieldPath: "spec.nodeName"
+            - name: "NRIA_FEATURES"
```

Comment:
Looking at this again makes me want to confirm: Does this disable the docker integration (`nri-docker`), or the docker plugin in the agent related to correlating process data and containers?
We'd like to disable both.

Reply:
Only the integration.